package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "start": "node ./bin/www",
     "dev": "nodemon ./bin/www",
-    "lint": "eslint controllers models routes"
+    "lint": "eslint ."
```

Comment:
I changed it to run on all files from directory root instead

Reply:
Nice.